bootstrap.py

```diff
@@ -32,23 +32,30 @@
 DEFAULT_VERSION = "1.4.2"
```

Comment:
This script is copied verbatim from somewhere else, let's add it to an black ignore file